### PR TITLE
Updated version for DownloadPackageV1 task (#16154)

### PR DESCRIPTION
**Pull request by aasim**
**Task name**: <Name of changed or new pipeline task>

**Description**: <Describe your changes here>

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DownloadPackageV1/multifilepackage.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class MultiFilePackage extends Package {
     private async getPackageFileContent(fileMetadata: any): Promise<Map<string, PackageFileResult>> {
         return new Promise<Map<string, PackageFileResult>>(resolve => {
             var resultMap = new Map<string, PackageFileResult>();
-            resultMap[fileMetadata.name] = new PackageFileResult(fileMetadata.protocolMetadata.data.Content, false);
```
Comment (satbai) [inline Tasks/DownloadPackageV1/multifilepackage.ts:52]: https://github.com/microsoft/azure-pipelines-tasks/pull/16154 are these file changes in a different PR for master?

File: Tasks/DownloadPackageV1/multifilepackage.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ export class MultiFilePackage extends Package {
                         if (filteredFileList.has(fileMetadatas[i].name)) {
                             const fileMetadata = fileMetadatas[i];
                             pkgFileUrlPromises.push(
-                                fileMetadata.protocolMetadata.data.StorageId != null
+                                fileMetadata.protocolMetadata.data.storageId != null
```
Comment (satbai) [inline Tasks/DownloadPackageV1/multifilepackage.ts:77]: Yay for JS being case sensitive

File: Tasks/DownloadPackageV1/multifilepackage.ts (hunk 1 above)
Comment (aasim) [reply Tasks/DownloadPackageV1/multifilepackage.ts:52]: Here is that change: https://github.com/microsoft/azure-pipelines-tasks/pull/16146

File: Tasks/DownloadPackageV1/multifilepackage.ts (hunk 2 above)
Comment (aasim) [reply Tasks/DownloadPackageV1/multifilepackage.ts:77]: and for onprem to have different casing than hosted :)

Comment (aasim) [reply Tasks/DownloadPackageV1/multifilepackage.ts:77]: we will have to make this fix before next onprem release goes out

**Review by satbai — APPROVED** [review]

**Pull request closed without merging**